packages/tenderly-core/src/internal/core/services/TenderlyService.ts

```diff
@@ -87,8 +89,8 @@ export class TenderlyService {
     return null;
   }
 
-  public async verifyContracts(request: TenderlyContractUploadRequest): Promise<void> {
-    logger.debug("Verifying contracts publicly.");
+  public async verifyContractsMultiCompiler(request: TenderlyVerifyContractsRequest): Promise<void> {
```

Comment:
`verifyContractsMultiCompiler` is a new function. Git marked it as if I changed the old function, but I didn't. Old function remained the same.